lib/samwise/client.rb

```diff
@@ -32,6 +40,18 @@ def excluded?(duns: nil)
 
     def small_business?(duns: nil)
       response = lookup_duns(duns: duns)
+      parse_response_for_small_business(response)
+    end
+
+    private
+
+    def parse_response_for_sam_status(response)
+      response.status == 200
+    end
+
+    def parse_response_for_small_business(response)
+      return false if response.code == 404
```

Comment:
I think I'm going to move more robust error handling to another story for now. This solution is good enough for now (e.g. if the response is not 200, let's just return false, since that's how the SAM.gov API works.